File: java/src/PerimetroArea.java
import java.util.Scanner;

public class PerimetroArea {

    public static void main(String[] args) {
        final double PI = 3.14159;
        Scanner scanner = new Scanner(System.in);
        int raio;
        double perim;
        double area;

        System.out.println("Entre com o valor do raio:");
        raio = scanner.nextInt();

        perim = 2 * PI * raio;

        area = PI * Math.pow(raio, 2);

        System.out.println("O perímetro da circunferência de raio " + raio + " é " + perim);

        System.out.println("A área é " + area);

        scanner.close();
    }
}

File: cpp/PerimetroArea.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    const double PI = 3.14159;
    int raio;
    double perim;
    double area;

    cout << "Entre com o valor do raio:" << endl;
    cin >> raio;

    perim = 2 * PI * raio;
    area = PI * pow(raio, 2);

    cout << "O perímetro da circunferência de raio " << raio << " é " << perim << endl;
    cout << "A área é " << area << endl;

    return 0;
}
